### ISSUE TITLE
Show reviews body text on other fediverse networks

### ISSUE THREAD

**Issue by winy98** [bookwyrm-social/bookwyrm#3412]
**Is your feature request related to a problem? Please describe.**
When I watch somebody's bookwyrm review from Mastodon, the post only shows a link to the review, not the body text (or almost an extract). But that doesn't happen with book comments. The comments are shown as text + link on Mastodon. 

**Describe the solution you'd like**
I think it'll be nice to show both, book reviews and comments, as text + link on the rest of fediverse networks.

**Comment by hughrun** [comment]
This is an issue at the other end: On BookWyrm Reviews are published as [`Article`](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-article)s, which vanilla Mastodon converts into [`Note`](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-note)s with a link.

It's up to each ActivityPub application how and whether they accept all Activity Vocabulary object types. [GoToSocial](https://github.com/superseriousbusiness/gotosocial) and the Mastodon fork [Hometown](https://github.com/hometown-fork/hometown), for example, do display Articles (and therefore reviews) in full.

**Comment by mouse-reeve** [comment]
I agree that philosophically it's an issue with other services, but I don't think there's a downside to changing them to federate as `Note` instead of `Article` since that is handled more gracefully in the wider network

**Comment by skolima** [comment]
That one is, uhm, rather disappointing to see - it makes it feel like Bookwyrm hardly federates at all, since the contents of the review are completely hidden from most other apps - there's not even a hint that the review is there! (I know, there's a link to the full contents - but nothing tells the viewer from Mastodon that there is anything more to see apart from the star rating - there's no incentive to click on that link).

And yes, 100% agree on the philosophical part, but unfortunately Mastodon now dominates the network, it also sets the de-facto standard behaviours.

**Comment by hughrun** [comment]
Mastodon is the app not supporting `Article`s and causing this issue — current BookWyrm behaviour is to send reviews as an Article and Mastodon's behaviour is to truncate and link Articles and present them locally as a `Note`.

Unfortunately Mastodon is the USA of ActivityPub so Mouse is probably right, the simplest thing to reduce confusion is to send reviews as a `Note` and assume they'll be published in full at the other end.

**Comment by suihkulokki** [comment]
> Mastodon is the app not supporting `Article`s and causing this issue — current BookWyrm behaviour is to send reviews as an Article and Mastodon's behaviour is to truncate and link Articles and present them locally as a `Note`.

> Unfortunately Mastodon is the USA of ActivityPub so Mouse is probably right, the simplest thing to reduce confusion is to send reviews as a `Note` and assume they'll be published in full at the other end.

I note (no pun intended) that wordpress federates their articles too as notes. So I don't see a big philosophical problem in making BookWyrm reviews notes as well.
